### DISCUSSION TITLE
Ring alarm using non-ring contact sensors

### DISCUSSION THREAD

**Discussion by Stevenlgts** [dgreif/ring#628] [Q&A] [answered]
I am considering getting the ring alarm system. My house was wired for a security system, so all the doors have wired contact sensors. I have all the wired sensors operating thru Homebridge RPi. Does anyone know if I can incorporate those wired sensors into a Ring alarm use this Ring plugin? I prefer to use the prewired sensors since I have 5 external doors, and they integrate into Homebridge perfectly.

**Comment by dgreif** [comment, marked as answer]
Unfortunately, there is no way to integrate other homebridge contact sensors with the Ring Alarm.  If you want those wired contact sensors to activate the Ring Alarm, you would need to purchase a Retrofit Kit: https://ring.com/products/alarm-security-kit-5-piece-retrofit-bundle.  I installed one of these for a family member and it worked well with their existing hard-wired sensors.  Each sensor (zone) will still show up in homebridge through this plugin, but they will be coming through the Ring Alarm instead of directly wired to the RPi.
